src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -964,6 +968,12 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, Component
                         translateValues: { index: index + 1 },
                     });
                 }
+                if (!(shortAnswerQuestion.solutions?.filter((solution) => solution.text!.trim().length >= this.maxLength).length === 0)) {
```

Comment:
```suggestion
                if (shortAnswerQuestion.solutions?.filter((solution) => solution.text!.trim().length >= this.maxLength).length !== 0) {
```
[nit] I think this makes the code a bit more readable.

Reply:
That's a good point. I adapted it.